resources/content/config-content/hosts/content-home/etc/cattle/dns/answers.json.ftl

```diff
@@ -18,6 +18,10 @@
                     </#if>
                 </#list>
         ],
+        	<#else>
+        "recurse": [
+            PARENT_DNS
```

Comment:
Remember we spoke how host container itself will get represented as "default" source in answers.json file? So for it in order to resolve external ips there should be an external DNS ref, therefore this line